pkg/gds/gds.go

```diff
@@ -113,9 +113,13 @@ func (s *GDS) Register(ctx context.Context, in *api.RegisterRequest) (out *api.R
 		VaspCategories:      in.VaspCategories,
 		EstablishedOn:       in.EstablishedOn,
 		Trixo:               in.Trixo,
-		VerificationStatus:  pb.VerificationState_SUBMITTED,
+		VerificationStatus:  pb.VerificationState_NO_VERIFICATION,
```

Comment:
This is also the zero value so this entire line could be safely omitted, but I like that you've added it here explicitly for clarity.